jenkins/Jenkinsfile.databricksnightly

```diff
@@ -78,7 +80,7 @@ pipeline {
                     sshagent(credentials : ['svcngcc_pubpriv']) {
                         sh "rm -rf spark-rapids-ci.tgz"
                         sh "tar -zcvf spark-rapids-ci.tgz *"
-                        sh "python3.6 ./jenkins/databricks/run-tests.py -c $CLUSTER_ID -z ./spark-rapids-ci.tgz -t $DATABRICKS_TOKEN -p /home/svcngcc/.ssh/id_rsa -l ./jenkins/databricks/build.sh -j $CI_RAPIDS_JAR -b $DATABRICKS_VERSION -k $SPARK_VERSION -a $SCALA_VERSION -f $CUDF_VERSION -u $CUDA_VERSION -m $CI_CUDF_JAR"
+                        sh "python3.6 ./jenkins/databricks/run-tests.py -z ./spark-rapids-ci.tgz -t $DATABRICKS_TOKEN -p /home/svcngcc/.ssh/id_rsa -l ./jenkins/databricks/build.sh -j $CI_RAPIDS_JAR -b $DATABRICKS_VERSION -k $SPARK_VERSION_TO_INSTALL_JARS -a $SCALA_VERSION -f $CUDF_VERSION -u $CUDA_VERSION -m $CI_CUDF_JAR -w $PUBLIC_KEY -o $CLUSTER_ID_LOC"
```

Comment:
Seems to me the output file is needed because we're overloaading run-tests to not only run tests on a cluster but also create the cluster.  Would it make more sense to have the Jenkins setup explicitly create the cluster first with a cluster creation script that returns its cluster ID as the lone thing to stdout (all other logging goes to stderr), and Jenkins tracks that cluster Id in a variable?  Admittedly, I'm not enough of a Jenkins expert to know if this makes sense.

Reply:
Yeah I think we can break it up that way and use env variable to pass between to get to the post step where it shuts down. I had a bit of problems with that but I think I read how to do it properly so I'll try it out.